src/app/features/users-page/users-page.component.html

```diff
@@ -1,13 +1,13 @@
 <a routerLink="/" class="back-link">Back To Home</a>
 <h1>List of all users:</h1>
-<table style="width: 1000px; border: 1px solid black">
+<table class="table-wrapper">
```

Comment:
bad class name
table wrapper is a wrapper for a table but in your case it is table itself not a wrapper

```
<div class="table-wrapper"><table></table></div>
```

rename to something like
`.user-table`